### DISCUSSION TITLE
Outside Temperature Stuck at 5 degrees

### DISCUSSION THREAD

**Discussion by Hansontech190** [klejejs/ha-thermia-heat-pump-integration#114] [General]
Since the 8th Dec my HA is showing outside temperature stuck at 5 degrees.  The heatpump interface on the unit itself is showing 9 degrees and so is the Thermia server but HA integration doesn’t seem to be matching.  Any ideas what this might be?
I’ve looked at the graph for Dec online and shows this:
![IMG_0128](https://github.com/user-attachments/assets/c6636779-b8f0-4199-a62b-a56c50a9b321)

HA Shows this:
![IMG_0129](https://github.com/user-attachments/assets/8c43dba4-30d2-4610-ad7b-c9a2fcb4fa77)

**Comment by MrNickIE** [comment]
Maybe related to https://github.com/klejejs/ha-thermia-heat-pump-integration/discussions/112 ?
